README.md

```diff
@@ -44,6 +38,27 @@ flutter:
           args: ['--quality=65', '--hint=graph', '--af']
 ```
 
+By default, the package runs the embedded precompiled cwebp binary. You can use the one specified in your `$PATH` by adding the `--from_path` flag. On how to install cwebp see [Optional cwebp installation](#optional_cwebp_installation).
+
+```yaml
+flutter:
+  assets:
+    - path: assets/logo.jpg
+      transformers:
+        - package: webp
+          args: ['--from_path']
+```
+
+## <a id="optional_cwebp_installation"></a> Optional cwebp installation
```

Comment:
Actually, all headers in Markdown files on GitHub (and pub) have identifiers (they use hyphens instead of underscores)

Reply:
But I'd say that if someone wants to use `cwebp` from their PATH they probably know how to install it in the first place. I'd remove this section and mention of it in the section above